xml/System.Drawing.Printing/PaperKind.xml

```diff
@@ -339,7 +339,7 @@
       </ReturnValue>
       <MemberValue>77</MemberValue>
       <Docs>
-        <summary>A4 rotated paper (297 mm by 210 mm). Requires Windows 98, Windows NT 4.0, or later.</summary>
+        <summary>A4 rotated paper (297 mm by 210 mm). Requires Windows NT 4.0 or later.</summary>
```

Comment:
Windows NT 4.0 was 25 years ago. I think it's safe to excise these references as well. :)

Reply:
In other words, we support it in all the operating system versions we currently support, so we can remove the requirement part?:

```suggestion
        <summary>A4 rotated paper (297 mm by 210 mm).</summary>
```

Reply:
Yes (assuming we're only talking about windows, and not Unix, of course).

Any remaining mention of "Windows NT" in the API docs can be removed, I think. If it's distinguishing from Unix, it should just say Windows.